stories/Modal.jsx

```diff
@@ -44,6 +45,13 @@ import {
 */
 import Sidebar from '../src/components/Sidebar';
```

Comment:
I think we should go for a Modal.Title component. Don't you think? We can't stick with the Sidebar one and I'm sure there will be some differences between the two titles behaviors.

Reply:
I do agree, but I will do it in a dedicated PR.
The `Modal` modifs here were only about using `Transition` instead of `CSSTransitionGroup`.

Reply:
Actually, I would go for a common set of `Title`, `Header`, `Footer` components, but in their dedicated folder.
The only difference between the behaviors I can think of is about style, but we can totally use `props.classNames` to differentiate. And any kind of `click` behavior can also be set by `props.onClick`.
We should discuss this in a dedicated issue.

Reply:
Yep you're right for the another-PR point. I did put a comment on the related issue. Thanks!